app/assets/javascripts/datepicker.js

```diff
@@ -1,16 +1,23 @@
 $(document).ready(function(){
-  $('[data-behaviour~=datepicker]').datepicker({
-    format: 'yyyy/mm/dd'
-  }).each(function (i, input){
+  $('[data-behaviour~=datepicker]').each(function() {
+    var picker = new Pikaday({
+      field: $(this)[0],
+      format: "YYYY/MM/DD",
```

Comment:
I will need to see it live - but I suspect uk users would be more comfortable with dd/mm/yyyy